### ISSUE TITLE
progress bar for rsync just stays at 0%

### ISSUE THREAD

**Issue by Wuodan** [pentoo/pentoo-installer#22]
progress bar for rsync (portage?) just sits at 0%
todo: check if this ever worked, if so, fix it, if not, ignore
low priority

**Comment by ZeroChaos-** [comment]
afaik it does work, it just stalls before starting (reading the file) and it done too fast to really update much

**Comment by ZeroChaos-** [comment]
I've installed a few hundred times lately, and it is working now for sure
